### DISCUSSION TITLE
can not connect to github

### DISCUSSION THREAD

**Discussion by henycave** [community/community#34491] [Other Feature Feedback, Questions, & Ideas] [answered]
every clone attempt i do ends up in an error like this:
fatal: unable to connect to github.com:
github.com[0: 140.82.121.3]: errno=Operation timed out

**Comment by MaroIsLife** [comment]
What method have you used to clone? 
Github removed unencrypted git protocol try cloning via HTTPS method or SSH

https://github.blog/2021-09-01-improving-git-protocol-security-github/

**Reply by henycave** [reply]
clone https://github.com/google/boringssl.git 
am trying to clone this repository
and it says 
fatal: unable to connect to github.com:
github.com[0: 140.82.121.4]: errno=Operation timed out

**Reply by GithubIsBroken** [reply]
i have a similar problem, when i try to log in with my details it says my account doesnt exist, when i try the forgotten password it says the email is not registered. i had to create a new account to post here.

**Comment by henycave** [comment, marked as answer]
I messed up my .gitconfig file a while ago while experimenting with it, so i deleted the file and try to connect with GitHub and it worked.

**Comment by Sumitmishra018** [comment]
Not able to connect with Github from Jenkins for multiple connection?
